### PR TITLE
PP-7093 DAC Audit - Dashboard Date range - remove auto reload

### PR DESCRIPTION
**Pull request by iqbalgds**
- When users change the data range select on the dashboard - they now have to press a `Update` button.
- The page does not auto reload when changing a value in the date range select.

*Screenshot*
image

### CODE REVIEW

File: app/views/dashboard/_activity.njk [hunk 1]
```diff
@@ -32,12 +36,10 @@
...
   }) }}
 
-  <noscript>
-    {{ govukButton({
-      classes: 'nojs-update-button',
-      text: "Update"
-    }) }}
-  </noscript>
+  {{ govukButton({
+    classes: 'dashboard-date-range__select',
```
Comment (sfount) [inline app/views/dashboard/_activity.njk:40]: Very minor but inconsistent use of `'` and `"` isn't ideal here

**Review by sfount — APPROVED** [review]
This looks like it resolves the issue it sets out to fix, looks good to me 👍 . Potentially the UX can be reviewed if this page is iterated on in the future

File: app/views/dashboard/_activity.njk (hunk 1 above)
Comment (iqbalgds) [reply app/views/dashboard/_activity.njk:40]: Thanks - I will update that now

File: app/views/dashboard/_activity.njk [hunk 2]
```diff
@@ -32,12 +36,10 @@
...
   }) }}
 
-  <noscript>
-    {{ govukButton({
-      classes: 'nojs-update-button',
-      text: "Update"
-    }) }}
-  </noscript>
+  {{ govukButton({
+    classes: "dashboard-date-range__select",
```
Comment (iqbalgds) [inline app/views/dashboard/_activity.njk:40]: Updated to double quotes

**Review by sfount — APPROVED** [review]

**Pull request merged**